### PR TITLE
Changed visibility of write boolean assets

### PR DESCRIPTION
**Pull request by ct-anaalbic**
Signed-off-by: ana.albic.comtrade <Ana.Albic@comtrade.com>

Brief description of the PR.
Visibility of write Boolean assets are changed.

**Related Issue**
This PR fixes issue #2340.

**Description of the solution adopted**
Value of assets is not visible if assets are Boolean and if they only have `write` permission. It looks like on screenshot bellow.

**Screenshots**
image

**Any side note on the changes made**
/

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolve issue #2340 - can one of the commiters check the code and merge when possible?

File: console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/assets/DeviceAssetsPanel.java [hunk 1]
```diff
@@ -369,6 +369,12 @@ public void handleEvent(BaseEvent be) {
             radioFalse.setValue(true);
             radioGroup.setOriginalValue(radioFalse);
         }
+        if(channel.getModeEnum().equals(GwtDeviceAssetChannelMode.WRITE)) {
+            radioTrue.setValue(null);
```
Comment (muros-ct) [inline console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/assets/DeviceAssetsPanel.java]: What means setting value to null?

Comment (ct-anaalbic) [reply console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/assets/DeviceAssetsPanel.java]: Hi @muros-ct.

It means if channel is only with write permission, than value in radio button is null, and user can not see value, but he can chose false or true option.

Comment (muros-ct) [reply console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/assets/DeviceAssetsPanel.java]: Ok, I understand now. It is disabling radio button functionality. None of buttons is selected, neither true or false. Hack.

**Pull request merged**